packages/react/tsconfig.json

```diff
@@ -17,6 +17,6 @@
     "module": "esnext",
     "target": "es2015"
   },
-  "exclude": ["node_modules", "src/**/*.test.ts", "src/**/*.test.tsx"],
+  "exclude": ["node_modules"],
```

Comment:
I'm not sure why we excluded tests from our tsconfig. But I guess it's fine not to exclude them.